app/assets/src/components/views/SampleView/Controls.jsx

```diff
@@ -69,8 +69,10 @@ class Controls extends React.Component {
           direction="left"
         />
       );
-    } else if (sample.status === "created" || !reportPresent) {
+    } else if (canEdit) {
```

Comment:
should we still check that there's no report present though? Right now, we only allow deletion if it's inconsequential because sample failed anyway